cypress/src/integration/integrations.spec.ts

```diff
@@ -15,13 +15,14 @@
  */
 /// <reference types="cypress" />
 import 'os';
+import { API_BASE_URL } from '../settings';
```

Comment:
we use `CYPRESS_baseUrl` as an override to set this so that we can run it against PR's. So I think we can revert this bit.